crates/forge/tests/integration/dispatchers.rs

```diff
@@ -576,7 +576,7 @@ fn nonexistent_method_call() {
     assert_case_output_contains!(
         result,
         "test_nonexistent_method_call",
-        "Entry point selector 0x01fdb214e1495025fa4baf660d34f03c0d8b5037cf10311d2a3202a806aa9485 not found in contract 0x00c0727f111a1f9fd7543cf36ae25feaa8184a9188daa4c9c148605445dcba9f"
+        "Entry point selector 0x01fdb214e1495025fa4baf660d34f03c0d8b5037cf10311d2a3202a806aa9485 not found in contract 0x0612c9ea6cdc79914ba9eae055225ee9df90368b02c9e463247d6c2d313000bb"
```

Comment:
What did cause that bump?

Reply:
I use the Scarb built locally for testing, it has a different compiler version